ui/src/shared/components/Dygraph.js

```diff
@@ -65,18 +71,26 @@ export default class Dygraph extends Component {
 
     const graphRef = this.graphRef
     const legendRef = this.legendRef
-    let finalLineColors = overrideLineColors
+    const finalLineColors = [...(overrideLineColors || LINE_COLORS)]
+
+    const hashColorDygraphSeries = {}
+    const {length} = finalLineColors
```

Comment:
currently the length of `finalLineColors` is 13, but what happens if this changes in the future?
as far as i understand, it's best to use a prime number as the modulo for hashing to avoid uneven distribution into buckets, so it may be safest to set this to 13 directly instead of `{length}`, and to add a comment above `LINE_COLORS` in `shared/graphs/helpers.js` to suggest keeping the number to 13, or that your hashing function is using the length as its modulo and that it should be kept at a prime length.